.travis.yml

```diff
@@ -25,23 +25,15 @@ env:
 
 matrix:
   allow_failures:
-    - env: distro=debian:10 playbook=test.yml
```

Comment:
why is this line removed? Is it related to the redis bug? If not I would leave it there and address it later if needed